common/lib/xmodule/xmodule/x_module.py

```diff
@@ -1214,6 +1213,7 @@ def descriptor_global_handler_url(block, handler_name, suffix='', query='', thir
     """
     raise NotImplementedError("Applications must monkey-patch this function before using handler_url for studio_view")
 
+descriptor_global_handler_url.unpatched = True
```

Comment:
Ah. Yeah, this is mildly evil. It's setting an attribute just so I can look for it later to see if it's been monkey patched in the interim.